.travis.yml

```diff
@@ -4,8 +4,7 @@ cache:
   npm: true
   ccache: true
 
-node_js:
-  - stable
```

Comment:
Why remove 'stable'?

Reply:
Stable is failing both in amd64 and ppc64le
https://travis-ci.org/github/genisysram/OpenCC/builds/746974897

Reply:
This seems a problem we should fix, rather than removing 'stable'.